### ISSUE TITLE
Alt text detection broken by Twitter

### ISSUE THREAD

**Issue by wmww** [aweary/twitter-required-alt-text#14]
Even after adding alt text, still says it's required and wont let you tweet. Observed just now on Firefox. I'll try Chrome and possibly make a fix later, if I remember.

**Comment by aweary** [comment]
Yeah I think Twitter updated something that broke this, will look into it soon

**Comment by aweary** [comment]
Fixed in https://github.com/aweary/twitter-required-alt-text/commit/9292c4cce9973b11b0ae121881ade8e0f933c17e and is pending review in the Chrome Web Store

**Comment by queerterpreter** [comment]
I realize this specific code is for Chrome, but do you know who's maintaining the Firefox extension and whether the update will also go through there?

**Comment by wmww** [comment]
I'm the publisher of the FF extension! I will test and submit it shortly.

**Comment by queerterpreter** [comment]
Oh, beautiful! Thank you (both) for all your work 💜

**Comment by wmww** [comment]
Firefox extension has been published and I've confirmed that v1.2 from the Mozilla store works for me. I'll leave closing this for aweary when the chrome extension releases.
